### ISSUE TITLE
aFieldValueIs only fires once with inline radio buttons

### ISSUE THREAD

**Issue by sturoid** [aldeed/meteor-autoform#983]
With 5.3.0 the problem was fixed with aFieldValueIs and checkboxes, but now I can't seem to get radios to fire more than once. This is what I have:

```
repeatEnds:
   type: String
   optional: true
   autoform:
      type: 'select-radio-inline'
      options: -> [{ label: 'Never', value: 'never' },{ label: 'After', value: 'after' }, { label: 'On', value: 'on' }]
```

```    
.row
   .col-md-4
      +afQuickField(name='repeatEnds')
   .col-md-8
      if afFieldValueIs name='repeatEnds' value='after'
         +afQuickField(name='repeatEndsAfter')
      if afFieldValueIs name='repeatEnds' value='on'
         +afQuickField(name='repeatEndsOn')
```

If I click on the 'after' or 'on', I can't make anything else fire after that. Am I doing something wrong here or is this a problem?

**Comment by cskaandorp** [comment]
+1

**Comment by udnaan** [comment]
+1

**Comment by rodcope1** [comment]
+1

**Comment by tcastelli** [comment]
+1

**Comment by cskaandorp** [comment]
Is this only an issue with inline radio's?? I have the same problem with radio buttons in general. The first time I use them, afFieldValueIs works, the second time it doesn't.

**Comment by sturoid** [comment]
I'm not sure. I have only done inline radios and single checkboxes. It very well could be though. I'll try it later today as well.

**Comment by tinchoz49** [comment]
Hey guys, same problem using the component "boolean-radios" !

**Comment by jmlv** [comment]
some problem +1

**Comment by udnaan** [comment]
Updating to the latest version fixed it for me.

**Comment by cskaandorp** [comment]
I am on the last version!!! It fires only once. As soon as I make a second choice in a list of radio buttons with a different conditional, it won't do anything anymore. Adnan are you really sure you tried with a couple of radio buttons and a couple of conditionals on them. Could you please confirm.. Thanks.

Casper

Sent from my iPhone

On 8 jun. 2015, at 21:56, Adnan <notifications@github.com<mailto:notifications@github.com>> wrote:


Updating to the latest version fixed it for me.

**Comment by tcastelli** [comment]
@udnaan which version are you using?
Still not working for me either on 5.3.0

**Comment by udnaan** [comment]
5.3.0 here as well. Perhaps it isn't an strictly an autoform issue. I changed and shuffled around a few packages and since then it has been working.

**Comment by jmlv** [comment]
Not working for me 5.3.0

**Comment by tinchoz49** [comment]
Same problem here, Not working in 5.3.0

**Comment by cskaandorp** [comment]
@udnaan, what packages did you change. By the way I am using Semantic-UI. Anybody else using the same thing. Since 5.2 was a disaster for me I automatically assumed this was an autoform issue.

**Comment by sturoid** [comment]
I'm not using Semantic UI. I just have bootstrap and the calendar picker as far as UI libraries.

**Comment by udnaan** [comment]
I made a test app [here](http://meteorpad.com/pad/Zyzezegrc2yMuJX6t/Leaderboard) and the issue is persistent. Oddly enough, it's gone in the production app. I quickly skimmed through the package list but couldn't find anything. Here is the version file for the app if it can be of any help:

accounts-base@1.2.0
accounts-facebook@1.0.4
accounts-google@1.0.4
accounts-oauth@1.1.5
accounts-password@1.1.1
accounts-ui-unstyled@1.1.7
alanning:roles@1.2.13
aldeed:autoform@5.3.0
aldeed:collection2@2.3.3
aldeed:simple-schema@1.3.3
aldeed:template-extension@3.4.3
arunoda:npm@0.2.6
autoupdate@1.2.1
base64@1.0.3
binary-heap@1.0.3
blaze@2.1.2
blaze-tools@1.0.3
boilerplate-generator@1.0.3
callback-hook@1.0.3
cfs:http-methods@0.0.29
check@1.0.5
chriswessels:hammer@3.1.1
chuangbo:cookie@1.1.0
cmather:handlebars-server@2.0.0
coffeescript@1.0.6
ctjp:meteor-intl-tel-input@3.7.1_2
dburles:mongo-collection-instances@0.2.6
dburles:spacebars-tohtml@1.0.0
ddp@1.1.0
deps@1.0.7
ejson@1.0.6
email@1.0.6
facebook@1.2.0
fastclick@1.0.3
fortawesome:fontawesome@4.3.0
fourseven:scss@2.1.1
geojson-utils@1.0.3
google@1.1.5
handlebars@1.0.3
html-tools@1.0.4
htmljs@1.0.4
http@1.1.0
id-map@1.0.3
iron:controller@1.0.7
iron:core@1.0.7
iron:dynamic-template@1.0.7
iron:layout@1.0.7
iron:location@1.0.7
iron:middleware-stack@1.0.7
iron:router@1.0.7
iron:url@1.0.7
jquery@1.11.3_2
json@1.0.3
launch-screen@1.0.2
lepozepo:s3@5.1.1
lepozepo:streams@0.2.0
less@1.0.14
livedata@1.0.13
localstorage@1.0.3
logging@1.0.7
matb33:collection-hooks@0.7.13
meteor@1.1.6
meteor-platform@1.2.2
meteorhacks:async@1.0.0
meteorhacks:fast-render@2.5.0
meteorhacks:inject-data@1.3.0
meteorhacks:npm@1.3.0
meteorhacks:picker@1.0.2
minifiers@1.1.5
minimongo@1.0.8
mobile-status-bar@1.0.3
momentjs:moment@2.8.4
mongo@1.1.0
mquandalle:bower@1.4.1
mquandalle:jade@0.4.3
mquandalle:jade-compiler@0.4.3
mrt:paypal@1.1.1
mrt:twilio-meteor@1.1.0
npm-bcrypt@0.7.8_2
npm-container@1.0.0
oauth@1.1.4
oauth2@1.1.3
observe-sequence@1.0.6
ongoworks:security@1.1.0
ordered-dict@1.0.3
random@1.0.3
reactive-dict@1.1.0
reactive-var@1.0.5
reload@1.1.3
retry@1.0.3
reywood:iron-router-ga@0.6.0
routepolicy@1.0.5
rubaxa:sortable@1.1.1
service-configuration@1.0.4
session@1.1.0
sewdn:collection-behaviours@0.2.0
sha@1.0.3
softwarerero:accounts-t9n@1.0.9
spacebars@1.0.6
spacebars-compiler@1.0.6
spiderable@1.0.7
srp@1.0.3
standard-app-packages@1.0.5
tap:i18n@1.4.1
templating@1.1.1
tracker@1.0.7
udnaan:autoform-selectize@0.1.4
udnaan:pretty-emails@0.0.3
udnaan:reactive-stripe@1.0.0
ui@1.0.6
underscore@1.0.3
url@1.0.4
useraccounts:bootstrap@1.11.1
useraccounts:core@1.11.1
webapp@1.2.0
webapp-hashing@1.0.3
zimme:active-route@2.0.3
zimme:iron-router-active@2.0.0_1

**Comment by cskaandorp** [comment]
I did the same thing, a very very simple form with the following packages:
aldeed:autoform     5.3.0
aldeed:collection2  2.3.3
coffeescript            1.0.6
meteor-platform     1.2.2

There was no joy. Radio group only fired once for afFieldValueIs. Both in dev -and- production. As far as I am concerned this is still an autoform issue.

**Comment by dkmooers** [comment]
Same issue here. afFieldValueIs only fires once on radios.

**Comment by spaeth** [comment]
Same to me.

**Comment by VladShcherbin** [comment]
+1

**Comment by jmlv** [comment]
Okay, we have a bug.

**Comment by VladShcherbin** [comment]
Same with the method `AutoForm.getFieldValue` - it fires only once and doesn't update reactively.

**Comment by cskaandorp** [comment]
I think there is also a 'smaller' issue with afFieldValueContains. It reacts sluggish, I sometimes have to check, uncheck and check again to make it do something...

**Comment by liuyork** [comment]
+1

**Comment by maxnowack** [comment]
+1

**Comment by nickfm** [comment]
+1
